config/routes.php

```diff
@@ -25,4 +25,11 @@
     '_locale' => implode('|', \App\EventSubscriber\getAvailableLanguages()),
 )));
 
+$routes->add('resources', new Route('/resources', array(
+    '_controller' => 'App\\Controller\\IndexController::resources',
+    '_locale' => 'en_GB',
+), array(
+    '_locale' => implode('|', \App\EventSubscriber\getAvailableLanguages()),
```

Comment:
You started from an old branch.. There is already a fix for translation in the about-page branch